app/views/mailers/administrator_notifications/channel_notifications_mailer/email_disconnect.liquid

```diff
@@ -0,0 +1,8 @@
+<p>Hello,</p>
+
+<p>Your Email Inbox configuration is not working. </p>
```

Comment:
Your email inbox has been disconnected due to configuration errors. Please update it to continue receiving messages in <brand_name>/Chatwoot.